console/src/components/MapCore/MapCore.js

```diff
@@ -365,6 +409,24 @@ class MapCoreBase extends React.Component {
 		this.carNode.style.setProperty('--map-orientation', orientation);
 	}
 
+	updateCarLayer = (location) => {
+		const newCarData = {
+			'type': 'FeatureCollection',
+			'features': [{
+				'type': 'Feature',
+				'geometry': {
+					'type': 'Point',
+					'coordinates': location.coordinates
+				}
+			}]
+		};
+
+		// update coordinates of the car
```

Comment:
I think we need a guard for whether `this.map.getSource(carLayerId)` exists around these 4 lines.